sources/web/datalab/static/dark.css

```diff
@@ -113,6 +113,16 @@ div.output {
 div.output_stderr {
   background-color: #a94442;
 }
+input.raw_input {
+  color: #ddd;
+  background-color: #333;
+}
+input.raw_input::selection {
+  background: #ccc;
+}
+input.raw_input::-moz-selection {
+  background: #999;
```

Comment:
I've pushed a new version using `rgba()` based on [this](http://stackoverflow.com/questions/7224445/css3-selection-behaves-differently-in-ff-chrome) StackOverflow post.

`::selection` is not in the officially CSS spec, but it was published in a [draft of Pseudo-Elements Level 4](https://www.w3.org/TR/css-pseudo-4/).

From [Mozilla Developer Network](https://developer.mozilla.org/en-US/docs/Web/CSS/::selection),

> Gecko is the only engine requiring the prefix. Due to the fact that the CSS parsing rules require dropping the whole rule when encountering an invalid pseudo-element, two separate rules must be written: ::-moz-selection, ::selection {...}. The rule would be dropped on non-Gecko browsers as ::-moz-selection is invalid on them.